examples/python-guide/Notebooks/Plot Example.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T00:15:02.683183Z",
+     "start_time": "2018-10-20T00:15:01.926205Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore') #suppress lgb userwarnings\n",
+    "\n",
+    "import lightgbm as lgb\n",
+    "import pandas as pd\n",
+    "import graphviz\n",
+    "\n",
+    "if lgb.compat.MATPLOTLIB_INSTALLED:\n",
+    "    import matplotlib.pyplot as plt\n",
+    "    %matplotlib inline\n",
+    "else:\n",
+    "    raise ImportError('You need to install matplotlib for plot_example.py.')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T00:15:02.688370Z",
+     "start_time": "2018-10-20T00:15:02.685240Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "lightgbm version: 2.2.1\n",
+      "pandas version: 0.23.3\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'lightgbm version: {lgb.__version__}')\n",
+    "print(f'pandas version: {pd.__version__}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load data..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T00:15:02.740998Z",
+     "start_time": "2018-10-20T00:15:02.690266Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "df_train = pd.read_csv('../regression/regression.train', header=None, sep='\\t')\n",
+    "df_test = pd.read_csv('../regression/regression.test', header=None, sep='\\t')\n",
```

Comment:
Paths should be updated, since notebooks is one level deeper than ordinary examples.